scripts/releaseUtils.ts

```diff
@@ -127,6 +132,12 @@ export function getVersionChoices(currentVersion: string) {
             title: 'major',
             value: inc('major')
           }
+        ]
+      : [
+          {
+            title: 'stable',
+            value: inc('patch')
+          }
         ]),
```

Comment:
This means that if we are in `alpha`, we can only go to `stable`, no?
I think we need to check if we are in `alpha` and add the `beta-minor` and `beta-major` option?

Reply:
Oh good point!

Reply:
Actually we only need a `beta` action, because we know from the `alpha` if we are doing a minor or major